tools/proofers/preview.inc

```diff
@@ -65,15 +65,16 @@ function get_preview_js_vars()
         "nowrap" => _("No wrap"),
     ];
 
-    $demo_string = "{$tag_names['t']} _{{$subscript}} <i>{$tag_names['i']}</i> ^{{$superscript}} <b>{$tag_names['b']}</b> <g>{$tag_names['g']}</g> <sc>{$tag_names['sc']}</sc> <f>{$tag_names['f']}</f>
```

Comment:
This would probably be a bit easier to read with a heredoc, eg:
```php
$demo_string = <<<DEMO
    {$tag_names['t']} ...
    DEMO;
```
php-cs-fixer will then indent the block which the interpreter will remove before building the string).